crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/stub_functions_trailing_comments.py

```diff
@@ -0,0 +1,25 @@
+# Regression tests for https://github.com/astral-sh/ruff/issues/11569
+
+
+# comment 1
+def foo(self) -> None: ...
+def bar(self) -> None: ...
+# comment 2
+
+# comment 3
+def baz(self) -> None:
+    return None
+# comment 4
+
+
+def foo(self) -> None: ...
+# comment 5
+
+def baz(self) -> None:
+    return None
+
+
+def foo(self) -> None:
+    ... # comment 5
+def baz(self) -> None:
+    return None
```

Comment:
nit: maybe also add a comment before end of file? I think that should also give the same behavior where the formatter will add two empty lines between the function (`baz`) and the comment.